scripts/pre_commit_linter.py

```diff
@@ -979,6 +982,26 @@ def _check_comments(all_files):
                         print '%s --> Line %s: %s' % (
                             filename, line_num + 1, message)
 
+                # Check that comment starts with a space.
+                if line.startswith('#') and len(line) > 1 and line[1] != ' ':
```

Comment:
A regex expression might be more suited here.

Reply:
Done, but if the goal was to reduce length of the conditional statement we haven't saved much length because it still needs to check if the line starts with # and the length is > 1 before it can match the regex because of comments like "#" where the line is only 1 character.

Reply:
Hi @ctao5660 you can reduce the length of conditional statement by checking all the three conditions with a single regex pattern something like this->`'^#[^\s].*$'`